tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -215,48 +218,63 @@ sub prepare_guest {
     my $vm = shift;
```

Comment:
Routine `prepare_guest` already exists so I recommend more specific name.
Also there is a lot of spaghetti code inside so I would create multiple smaller routines.

Reply:
It's not easy to split into multiple routines, because they mainly deal with different situations. But the name should be changed. It has been changed to prepare_ guest_ for_ sriov_ passthrough。